### ISSUE TITLE
Evaluate if `noImplicitAny` is worth turning on in `tsconfig`

### ISSUE THREAD

**Issue by thehenrytsai** [TBD54566975/dwn-sdk-js#298]
Upon reviewing and fixing lingering TS errors in PR #293 intended to complete issue #261, it has occurred to me that the current codebase depends on implicit `any` type in various places, I have made a few attempts to augment the code with `(foo as any).bar`, but it starts to feel either unnecessarily verbose, harder to read, and/or hacky.

Thus I am turning off the `noImplicitAny` rule for now to unblock the completion of #293 and #261 and treat this as a separate issue to investigate and evaluate.

**Comment by thehenrytsai** [comment]
Rule is enabled again with all triggered errors fixed.
